tests/test_mcp_toolset.py

```diff
@@ -0,0 +1,91 @@
+from holmes.core.tools import (
+    ToolParameter,
+)
+from mcp.types import ListToolsResult, Tool
+from holmes.plugins.toolsets.mcp.toolset_mcp import MCPToolset, MCPTool
+
+
+def test_parse_mcp_tool_input_schema_conversion():
+    input_schema_sample = {
+        "type": "object",
+        "properties": {
+            "symbol": {"type": "string"},
+            "qty": {"type": "integer", "description": "example for description"},
+            "side": {
+                "type": "string",
+                "enum": ["buy", "sell"],
+            },  # find more examples to improve description with format hints.
+            "limit_price": {"type": "number"},
+        },
+        "required": ["symbol", "qty", "side"],
+    }
+
+    expected_ouput = {
+        "symbol": ToolParameter(type="string", required=True),
+        "qty": ToolParameter(
+            type="integer", required=True, description="example for description"
+        ),
+        "side": ToolParameter(type="string", required=True),
+        "limit_price": ToolParameter(type="number", required=False),
+    }
+    tool_params = MCPTool.parse_input_schema(input_schema_sample)
+
+    assert tool_params == expected_ouput
+
+
+def test_parse_mcp_tool():
```

Comment:
Consider to join this test with the one above. Not sure that how this is currently implelemtned gives any value. (We only checking part of the field and not all of them)
Also the tool should create the parms so it feels like it should be joined. WDYT?